clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -21,23 +21,30 @@
 
 import static org.apache.gravitino.client.GravitinoClientBase.Builder;
 
+import org.apache.gravitino.cli.GravitinoConfig;
+import org.apache.gravitino.cli.OAuthData;
 import org.apache.gravitino.cli.outputs.PlainFormat;
 import org.apache.gravitino.cli.outputs.TableFormat;
+import org.apache.gravitino.client.DefaultOAuth2TokenProvider;
 import org.apache.gravitino.client.GravitinoAdminClient;
 import org.apache.gravitino.client.GravitinoClient;
 import org.apache.gravitino.exceptions.NoSuchMetalakeException;
 
 /* The base for all commands. */
 public abstract class Command {
-  private final String url;
-  private final boolean ignoreVersions;
-  private final String outputFormat;
-  public static String OUTPUT_FORMAT_TABLE = "table";
-  public static String OUTPUT_FORMAT_PLAIN = "plain";
+  public static final String OUTPUT_FORMAT_TABLE = "table";
+  public static final String OUTPUT_FORMAT_PLAIN = "plain";
 
   protected static String authentication = null;
   protected static String userName = null;
 
+  private static final String SIMPLE_AUTH = "simple";
+  private static final String OAUTH_AUTH = "oauth";
```

Comment:
This is very minor: about the naming convension, will "AUTH_SIMPLE" /  "AUTH_OAUTH" be better than "SIMPLE_AUTH" / "OAUTH_AUTH"? As in the above, I see "OUTPUT_FORMAT_TABLE" and "OUTPUT_FORMAT_PLAIN", they share the same prefix.

Reply:
I went this was as other code uses "SIMPLE_AUTH_TYPE" I don't mind either way.

Reply:
okay